src/lib/SCEditor.js

```diff
@@ -622,6 +622,7 @@ export default function SCEditor(original, userOptions) {
 		if (form) {
 			dom.on(form, 'reset', handleFormReset);
 			dom.on(form, 'submit', base.updateOriginal, dom.EVENT_CAPTURE);
+			dom.on(window, 'unload', base.updateOriginal, dom.EVENT_CAPTURE);
```

Comment:
There's always a textarea so we don't need to check if the editor is within a form.

When testing this I've noticed restoring is not 100% reliable in Chrome with either the `unload` or `beforeunload` events. Chrome seems to, at least sometimes, not change the value of the textarea to the previously saved value until after the editor has been created.

Adding a `pageshow` event that loads the value from the textarea seems to fix the issue as Chrome always appears to have set the textures value to the previously saved one by the time that's triggered. To avoid issues we would need to also use the `pagehide` event to save the value to the textarea.

This seems to be the most reliable from testing so far:
```js
dom.on(window, 'pagehide', base.updateOriginal);
dom.on(window, 'pageshow', handleFormReset);
```